ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -106,59 +127,130 @@ Control {
-                visible: count
-                ScrollBar.vertical: StatusScrollBar { }
-                delegate: StatusListItem { // TODO custom delegate
-                    width: GridView.view.cellWidth - Style.current.smallPadding
-                    height: GridView.view.cellHeight - Style.current.smallPadding
-                    title: model.name ?? ""
-                    statusListItemTitle.font.pixelSize: 17
-                    statusListItemTitle.font.bold: true
-                    subTitle: model.description ?? ""
-                    tertiaryTitle: qsTr("%n member(s)", "", model.membersCount ?? 0)
-                    asset.name: model.image ?? model.name ?? ""
-                    asset.isImage: asset.name.startsWith(Constants.dataImagePrefix)
-                    asset.isLetterIdenticon: !model.image
-                    asset.color: model.color ?? ""
-                    asset.width: 40
-                    asset.height: 40
-                    border.width: 1
-                    border.color: Theme.palette.baseColor2
-                    loading: !model.id
-                    components: [
-                        StatusIcon {
-                            visible: !!model.memberRole &&
-                                     model.memberRole === Constants.memberRole.owner ||
-                                     model.memberRole === Constants.memberRole.admin ||
-                                     model.memberRole === Constants.memberRole.tokenMaster
-                            anchors.verticalCenter: parent.verticalCenter
-                            icon: "crown"
-                            color: Theme.palette.directColor1
+                Layout.topMargin: (communitiesView.contentY > Style.current.padding) ? 1 : Style.current.padding
+                Behavior on Layout.topMargin { NumberAnimation { duration: 50 } }
+                clip: true
+                StatusGridView {
+                    id: communitiesView
+                    width: 606
+                    height: parent.height
+                    anchors.top: parent.top
+                    anchors.topMargin: Style.current.halfPadding
+                    anchors.bottom: parent.bottom
+                    anchors.bottomMargin: Style.current.halfPadding
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    anchors.horizontalCenterOffset: Style.current.halfPadding
+                    clip: false
+                    cellWidth: d.delegateWidthM
+                    cellHeight: d.delegateHeightM
+                    visible: count
+                    model: communitiesProxyModel
+                    ScrollBar.vertical: StatusScrollBar { }
+                    delegate: StatusCommunityCard {
+                        id: profileDialogCommunityCard
+                        readonly property var permissionsList: model.permissionsModel //TODO: Add permissions model in the community model
```

Comment:
The `permissionsMode` is missing from the communities model. Adding a task to add it in nim